modules/iam-read-only-policy/versions.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.15.0"
```

Comment:
Same version requirements here as in the example.

Reply:
done